flaskbb/cli/plugins.py

```diff
@@ -128,3 +128,29 @@ def list_plugins():
             click.secho("    - {} (version {})".format(
                 plugin.name, plugin.version), bold=True
             )
+
+
+@plugins.command("migrations")
+@click.argument("plugin_identifier")
+def migrate_plugin(plugin_identifier):
+    """Installs a new plugin."""
```

Comment:
"Perform database migrations for a plugin" or something a long the lines would be better